packages/eslint-plugin-pf-codemods/lib/rules/wizard-remove-props.js

```diff
@@ -0,0 +1,35 @@
+const { getPackageImports } = require('../helpers');
+
+const renames = {
```

Comment:
This could just be a list, but since `Boolean('REMOVE') == true`, you're okay :)